core/services/fluxmonitorv2/helpers_test.go

```diff
@@ -25,3 +25,13 @@ func (fm *FluxMonitor) ExportedRoundState() {
 func (fm *FluxMonitor) ExportedRespondToNewRoundLog(log *flux_aggregator_wrapper.FluxAggregatorNewRound, broadcast log.Broadcast) {
 	fm.respondToNewRoundLog(*log, broadcast)
 }
+
+func (fm *FluxMonitor) ExportedRespondToFlagsRaisedLog() {
+	fm.respondToFlagsRaisedLog()
+	fm.rotateSelectLoop()
+}
+
+func (fm *FluxMonitor) rotateSelectLoop() {
+	// the PollRequest is sent to 'rotate' the main select loop, so that new timers will be evaluated
+	fm.pollManager.chPoll <- PollRequest{Type: PollRequestTypeUnknown}
```

Comment:
This should use a new type otherwise it could get confusing in the future

Reply:
hmm i'm relying on PollRequestTypeUnknown breaking that loop - i could add another type with the same behavior i guess